watson_developer_cloud/conversation_v1.py

```diff
@@ -312,55 +529,81 @@ def create_example(self, workspace_id, intent, text):
 
         Add a new user input example to an intent.
 
-        :param workspace_id: The workspace ID.
-        :param intent: The intent name (for example, `pizza_order`).
-        :param text: The text of a user input example.
+        :param str workspace_id: The workspace ID.
+        :param str intent: The intent name (for example, `pizza_order`).
+        :param str text: The text of a user input example.
+        :return: A `dict` containing the `Example` response.
+        :rtype: dict
         """
+        if workspace_id is None:
+            raise ValueError('workspace_id must be provided')
+        if intent is None:
+            raise ValueError('intent must be provided')
+        if text is None:
+            raise ValueError('text must be provided')
         params = {'version': self.version}
         data = {'text': text}
-        return self.request(
+        response = self.request(
             method='POST',
             url='/v1/workspaces/{0}/intents/{1}/examples'.format(
                 workspace_id, intent),
             params=params,
             json=data,
             accept_json=True)
+        return response
 
     def delete_example(self, workspace_id, intent, text):
         """
         Delete user input example.
 
         Delete a user input example from an intent.
 
-        :param workspace_id: The workspace ID.
-        :param intent: The intent name (for example, `pizza_order`).
-        :param text: The text of the user input example.
+        :param str workspace_id: The workspace ID.
+        :param str intent: The intent name (for example, `pizza_order`).
+        :param str text: The text of the user input example.
+        :rtype: None
         """
+        if workspace_id is None:
+            raise ValueError('workspace_id must be provided')
+        if intent is None:
+            raise ValueError('intent must be provided')
+        if text is None:
+            raise ValueError('text must be provided')
         params = {'version': self.version}
-        return self.request(
+        self.request(
             method='DELETE',
             url='/v1/workspaces/{0}/intents/{1}/examples/{2}'.format(
                 workspace_id, intent, text),
             params=params,
             accept_json=True)
+        return None
```

Comment:
We should return the empty object, not `None`, so that users can always try to display the response without it blowing up.

Reply:
Happily, there is no problem with trying to print `None`.  Python handles it just fine without any exceptions:
```
[mkistler@mkistlers-MacBook-Pro] ~>python
Python 2.7.12 (default, Oct 11 2016, 05:20:59) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> response = None
>>> try:
...     print(json.dumps(response, indent=2)) 
... except:
...     print("exception!")
... 
null
>>> 
```
If some other version of Python did not handle this as gracefully, that is easily fixed:
```
>>> print(json.dumps(response or {}, indent=2)) 
{}
>>> 
```